platform/windows/detect.py

```diff
@@ -269,12 +269,14 @@ def configure_msvc(env, manual_msvc_config):
         "dwmapi",
     ]
 
-    env.AppendUnique(CPPDEFINES=["VULKAN_ENABLED"])
-    if not env["use_volk"]:
-        LIBS += ["vulkan"]
-
-    env.AppendUnique(CPPDEFINES=["GLES3_ENABLED"])
-    LIBS += ["opengl32"]
+    if env["vulkan"]:
```

Comment:
ok I implemented this because there are switches in the scons file to turn these on and off, but then they were being ignored on windows specifically, because of the use of the detect script